src/fairseq2/nn/padding.py

```diff
@@ -181,6 +181,8 @@ def get_seqs_and_padding_mask(
     seq_lens = data["seq_lens"]
 
     if device is not None:
+        if not isinstance(seq_lens, Tensor):
```

Comment:
fs2n vs fs2 inconsitency hack solving

Reply:
Yep, already verified to be stable in returning lists instead of tensors.